### PR TITLE
simplistic search test

### PR DESCRIPTION
**Pull request by kkraune**
- add a simple search example for completeness

### CODE REVIEW

File: vespa-cloud/vespa-documentation-search/src/test/java/ai/vespa/cloud/docsearch/VespaDocSystemTest.java [hunk 1]
```diff
@@ -111,11 +129,19 @@ private void removeTestDoc(String id) {
 
     public String getTestDoc(String id) {
         HttpResponse<String> res = testEndpoint.send(testEndpoint
-                .request("/document/v1/open/doc/docid/" + id.split(":")[4]) // id:open:doc::open/documentation/annotations.html
+                .request("/document/v1/open/doc/docid/" + id.split(":")[4])
```
Comment (jonmv) [inline vespa-cloud/vespa-documentation-search/src/test/java/ai/vespa/cloud/docsearch/VespaDocSystemTest.java:132]: URLEncoder.encode all parts here, or at the very least the user speicified?

Comment (jonmv) [reply vespa-cloud/vespa-documentation-search/src/test/java/ai/vespa/cloud/docsearch/VespaDocSystemTest.java:132]: (This is something many users have gotten wrong previously.)

**Review by jonmv — APPROVED** [review]
Nice!

File: vespa-cloud/vespa-documentation-search/src/test/java/ai/vespa/cloud/docsearch/VespaDocSystemTest.java (hunk 1 above)
Comment (kkraune) [reply vespa-cloud/vespa-documentation-search/src/test/java/ai/vespa/cloud/docsearch/VespaDocSystemTest.java:132]: good point! will add

**Pull request merged**